### PR TITLE
updated README with pnpm

### PR DESCRIPTION
**Pull request by shikharish**
`npm` -> `pnpm`

### CODE REVIEW

**Comment by rajivharlalka** [comment]
IMO should give alternatives too, `yarn/npm`

**Comment by rajivharlalka** [comment]
The JS community is so fragile there would be newer package manager that is blazing fast within a year, and this would become stale again, hence having alternatives help.

**Comment by harshkhandeparkar** [comment]
> IMO should give alternatives too, `yarn/npm`

We can't give alternatives because the lockfile is in pnpm.

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
 To run Chillzone front-end on your local system
 
-1. Make sure you have `npm` installed on your system.
+1. Make sure you have `pnpm` installed on your system.
```
Comment (harshkhandeparkar) [inline README.md:54]: Can you add a link to installation instructions?

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Review by harshkhandeparkar — APPROVED** [review]

**Pull request merged**